test/test_sparse_csr.py

```diff
@@ -183,6 +189,7 @@ def test_coo_to_csr_convert(self, device, dtype, coalesced):
 
     @onlyCPU
     @dtypes(torch.float, torch.double)
+    @unittest.skipIf(IS_MACOS or IS_WINDOWS, "MKL doesn't work on windows or mac")
```

Comment:
this skip test is the only one needed due to is the only one expecting and MKL related warning